sirepo/package_data/static/js/elegant.js

```diff
@@ -1443,3 +1443,81 @@ SIREPO.app.directive('srBunchEditor', function(appState, panelState) {
         },
     };
 });
+
+
+SIREPO.app.directive('viewLogIframe', function(appState, requestSender) {
+    return {
+        restrict: 'A',
+        scope: {
+
+        },
+        // TODO: At error case for when log cannot be found, edit case below
+        template: `
+            <a href data-ng-click="viewLog()">View Log</a>
+            <div class="modal fade" id="sr-iframe-text-view" tabindex="-1" role="dialog">
+              <div class="modal-dialog modal-lg">
+                <div class="modal-content">
+                  <div class="modal-header bg-warning">
+                    <span class="lead modal-title text-info">Log</span>
+                    <div class="sr-panel-options pull-right">
+                      <a data-ng-href="{{ downloadLog() }}" target="_blank">
+                        <span class="sr-panel-heading glyphicon glyphicon-cloud-download" style="margin-bottom: 0"></span>
+                      </a>
+                      &nbsp&nbsp
+                      <button type="button" class="close" data-dismiss="modal">
+                        <span>&times;</span>
```

Comment:
Rather than the nbsp nbsp, you could add a style="margin-left: 1ex" to the close button.